ocs_ci/krkn_chaos/background_cluster_operations.py

```
@@ -156,12 +160,88 @@ def __init__(
+        env_data = config.ENV_DATA
+        if env_data.get("krkn_config"):
+            return env_data.get("krkn_config", {}).get(
+                "background_cluster_operations", {}
+            )
+        if env_data.get("resiliency_config"):
+            return env_data.get("resiliency_config", {}).get(
+                "background_cluster_operations", {}
+            )
+        return {}
+
+    def _apply_cephx_keyrotation_config(self):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `_apply_cephx_keyrotation_config`

Add return type annotation: `None`

(ANN202)